core/templates/dev/head/pages/exploration_player/StatsReportingService.js

```diff
@@ -66,6 +105,28 @@ oppia.factory('StatsReportingService', [
       },
       // Note that this also resets the stateStopwatch.
       recordExplorationStarted: function(stateName, params) {
+        if (ENABLE_NEW_STATS_FRAMEWORK) {
+          eventDicts.push({
+            event_type: STATS_EVENT_TYPES.EVENT_TYPE_START_EXPLORATION,
+            params: params,
+            session_id: sessionId,
+            state_name: stateName,
+            version: explorationVersion
+          });
+
+          eventDicts.push({
+            event_type: STATS_EVENT_TYPES.EVENT_TYPE_STATE_HIT,
+            client_time_spent_in_secs: 0.0,
+            old_params: params,
+            session_id: sessionId,
+            new_state_name: stateName,
+            exploration_version: explorationVersion,
+            is_first_hit: true
```

Comment:
Does the backend actually use this? Ditto for other occurrences of is_first_hit below.

Reply:
It used to. Redundant now.